harmony/harmony.py

```diff
@@ -260,7 +272,7 @@ def _status_url(self, job_id: str) -> str:
 
     def _params(self, request: Request) -> dict:
         """Creates a dictionary of request query parameters from the given request."""
-        params = {}
+        params = {'forceAsync': True}
```

Comment:
Looks like this means users cannot override to allow sync requests - is that true, and is that what we want?

Reply:
That is true. They can get sync-like behavior by submitting the request followed by running any of the following blocking method:
- wait_for_processing()
- result_json()
- result_urls()

It'd also be possible to alter the behavior of submit() to optionally block right away. What I was trying to do is present a sync or async experience through `harmony-py` while at the same time as having a consistent back-end workflow between `harmony-py` and Harmony.